### ISSUE TITLE
lazy load on `vim.notify`

### ISSUE THREAD

**Issue by ranjithshegde** [vigoux/notifier.nvim#18]
Since there is no discussions section in this repo, I am asking a question as an issue, so apologies in advance!

I am trying to switch to this plugin from `nvim-notify`, not a fan of popups!

I use the following somewhere in my config:  `vim.notify = require("notify")` when i use `nvim-notify`. That way I can lazy load `nvim-notify` with packer using `module = "notify"`, and have it loaded on `vim.notify` calls.

How do I do something similar with this plugin? I can of course do `module = "notifier"` but that does not allow it to be lazy loaded when some function calls `vim.notify`.

Basically any method that allows me to lazy load this plugin on `vim.notify` calls?

**Comment by folke** [comment]
You could do something like:

```lua
vim.notify = function(...)
   require("notifier").notify(...)
end
```

**Comment by vigoux** [comment]
Yup, recent change brought by @folke, that should work.

**Comment by ranjithshegde** [comment]
Thanks for your responses! I tried @folke's suggestion. While it does work for `vim.notify` calls, I miss the `[$/progress]`.
That was the difference with `nvim-notify`. Because it was the user who was calling `progress` using notify, it loaded on demand.

I suppose I will just not lazy load. 

Thanks for your help!

**Comment by vigoux** [comment]
The plugin _should_ load pretty fast as it is really lightweight.

There's no plan for lazyloading though in the plugin, sorry !
